syntax = "proto3";

package helloworld;

service HelloWorldService {
    rpc SayHello (HelloRequest) returns (HelloResponse);
    rpc ReceiveHello (HelloMessage) returns (AckMessage);
}

message HelloRequest {
    string message = 1;
}

message HelloResponse {
    string message = 1;
}

message HelloMessage {
    string message = 1;
}

message AckMessage {
    string message = 1;
}
